tools/run_tests/performance/scenario_config.py

```diff
@@ -1025,6 +1052,7 @@ def __str__(self):
     'node_express': NodeExpressLanguage(),
     'ruby' : RubyLanguage(),
     'php' : PhpLanguage(),
+    'php_ext' : PhpLanguage_ext(),
```

Comment:
I'd use a more descriptive name (and also one that complies to python code style for naming classes).
PhpWithProtobufCLanguage seems totally fine.

Reply:
You can also reuse the same class:
Something in this sense:
'php' : PhpLanguage(),
'php_XXX' : PhpLanguage(use_protobuf_c_extension=True),

Reply:
This idea sounds cool! It can re-use almost all the code. 
I updated the code. Now it has 2 languages: `php_protobuf_php` and `php_protobuf_c`. By passing `use_protobuf_c_extension=True`, it can also be used to generate different scenario name.